# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/dns/TRR.h
# Commit: c543a3a008fa
# Full Hash: c543a3a008fa894308984c95591efa695ed49221
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2020-09-15 09:29:30
# Regressor Bug: 1645108
# File Overlap Count: 2
# Description:
#   Bug 1645108 - Parse additional section of TRR response r=dragana,necko-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87088
# ==============================================================================

diff -r 8e50285987db -r c543a3a008fa netwerk/dns/TRR.h
--- a/netwerk/dns/TRR.h	Mon Sep 14 21:12:52 2020 +0000
+++ b/netwerk/dns/TRR.h	Mon Sep 14 21:41:44 2020 +0000
@@ -9,6 +9,7 @@
 
 #include "mozilla/net/DNSByTypeRecord.h"
 #include "mozilla/Assertions.h"
+#include "nsClassHashtable.h"
 #include "nsIChannel.h"
 #include "nsIHttpPushListener.h"
 #include "nsIInterfaceRequestor.h"
@@ -155,6 +156,8 @@
   nsresult FollowCname(nsIChannel* aChannel);
 
   bool UseDefaultServer();
+  void SaveAdditionalRecords(
+      const nsClassHashtable<nsCStringHashKey, DOHresp>& aRecords);
 
   nsresult CreateChannelHelper(nsIURI* aUri, nsIChannel** aResult);
 